### ISSUE TITLE
Paypal asks to integrate new API

### ISSUE THREAD

**Issue by cahueya** [concrete5-community-store/community_store_paypal_standard#18]
A customer of mine who uses the Paypal Payment Gateway (v 1.2.2) has been contacted by Paypal, the mail looks legit.

The paypal contact asks to integrate the most recent API of Paypal and linked to their docs site:
[https://developer.paypal.com/docs/checkout/standard/integrate/](https://developer.paypal.com/docs/checkout/standard/integrate/)

Is this on the roadmap?

Thank you!

**Comment by Mesuva** [comment]
Yes, this has been around for a while - https://github.com/concretecms-community-store/community_store_paypal_checkout
